configurations/alternator/streams-with-nemesis.yaml

```diff
@@ -0,0 +1,5 @@
+nemesis_class_name: 'NemesisSequence'
```

Comment:
why running this specific nemesis and not other nemesis?

Reply:
Want to pick a specific set, that for sure would have topology change in it.

I run this only once but didn't yet looked into the result.